great_expectations/expectations/window.py

```diff
@@ -21,8 +21,11 @@ class Window(pydantic.BaseModel):
 
     constraint_fn: str
     parameter_name: str
-    range: int
-    offset: Offset
+    range: int | None = Field(
+        default=None,
+        description="The number of previous invocations to consider. If None, all invocations will be considered.",
+    )
+    offset: Offset | None = None
```

Comment:
Can you add a description here also?

Reply:
We may need to add some validation? That is, if one is None but not the other, does that make sense?